Xamarin.Forms.Platform.iOS/Renderers/ShellRenderer.cs

```diff
@@ -255,5 +261,26 @@ void SetupCurrentShellItem()
 				OnCurrentItemChanged();
 			}
 		}
+
+		public class DesignerFlyoutRenderer : IShellFlyoutRenderer
```

Comment:
Does it have to be public?

Reply:
it has to be internal or better it looks like